content/docs/2.12/scalers/gcp-cloud-tasks.md

```diff
@@ -0,0 +1,151 @@
++++
+title = "Google Cloud Platform‎ Cloud Tasks"
+availability = "v2.12+"
+maintainer = "Community"
+description = "Scale applications based on Google Cloud Platform‎ Cloud Tasks."
+go_file = "gcp_cloud_tasks_scaler"
++++
+
+### Trigger Specification
+
+This specification describes the `gcp-cloudtasks` trigger for Google Cloud Platform‎ Cloud Tasks.
+
+```yaml
+triggers:
+- type: gcp-cloudtasks
+  metadata:
+    value: "5" # Optional - Default is 100
+    activationValue: "10.5" # Optional - Default is 0
+    queueName: "myqueue" # Required
+    projectID: "myproject" # Required, the project where the queue resides
+    credentialsFromEnv: GOOGLE_APPLICATION_CREDENTIALS_JSON # Required
+```
+
+The Google Cloud Platform‎ (GCP) Cloud Tasks trigger allows you to scale based on the number of tasks queued in you queue.
+
+The `credentialsFromEnv` property maps to the name of an environment variable in the scale target (`scaleTargetRef`) that contains the service account credentials (JSON). KEDA will use those to connect to Google Cloud Platform and collect the required stack driver metrics in order to read the number of messages in the Cloud Task queue.
```

Comment:
Let's align this with the list below

Reply:
What do you mean exactly by this? Sorry, I do not understand it.

Reply:
Yeah, please put that down into the list next to the other properties.